spec/factories/medical_histories.rb

```diff
@@ -23,6 +23,14 @@
       diagnosed_with_hypertension { MedicalHistory::MEDICAL_HISTORY_ANSWERS[:unknown] }
     end
 
+    trait :hypertension do
```

Comment:
Consider adjectives, `hypertensive`, `diabetic`?

Reply:
I considered that. I thought it might get confusing because `hypertensive` and `diabetic` means something else in our system: high BP, high BS etc.

What do you think?

Reply:
However, on second-thought, Diagnosis is pretty canonical at this point.

Reply:
How about `hypertension_yes` and `diabetes_yes`?

Reply:
@prabhanshuguptagit I'm not a big fan of the `yes` suffix. I'd say the original `hypertension` `diabetes` naming was better. I'll let you take a final call either way 👍